### PR TITLE
FCM Firelog to clearcut migration

### PR DESCRIPTION
**Pull request by eldhosembabu**
FCM Firelog to clearcut migration (b/295010819)

### CODE REVIEW

File: FirebaseMessaging/Sources/FIRMessagingExtensionHelper.m [hunk 1]
```diff
@@ -205,7 +205,7 @@ - (void)deliverNotification {
 - (void)exportDeliveryMetricsToBigQueryWithMessageInfo:(NSDictionary *)info {
   GDTCORTransport *transport = [[GDTCORTransport alloc] initWithMappingID:@"1249"
```
[the remark this answers is not in the document]
Comment (eldhosembabu) [reply FirebaseMessaging/Sources/FIRMessagingExtensionHelper.m:206]: I believe it should remain the same. I'll double check and get back.

[the remark this answers is not in the document]
Comment (miketaylor24601) [reply FirebaseMessaging/Sources/FIRMessagingExtensionHelper.m:206]: I double checked the configurations and as far as I can tell the two backends are consistent.

File: FirebaseMessaging/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+#Unversioned
```
Comment (ncooke3) [inline FirebaseMessaging/CHANGELOG.md:1]: The release tooling will look for `# Unreleased` when populating CHANGELOG entry versions.

```suggestion
# Unreleased
```

File: FirebaseMessaging/Sources/FIRMessagingExtensionHelper.m (hunk 1 above)
[the remark this answers is not in the document]
Comment (ncooke3) [reply FirebaseMessaging/Sources/FIRMessagingExtensionHelper.m:206]: Sounds good, thanks @eldhosembabu and @miketaylor24601

**Review by ncooke3 — APPROVED** [review]
LGTM on green CI

**Pull request merged**